examples/client_test.cpp

```diff
@@ -30,7 +30,6 @@ POSSIBILITY OF SUCH DAMAGE.
 
 */
 
-#include <cstdio> // for snprintf
```

Comment:
`std::snprintf()` is used in this file. It seems inappropriate to remove the include and instead rely on some other header including this "by accident".